extensions/packageManager/src/node/index.ts

```diff
@@ -9,6 +9,41 @@ import { SchemaMerger } from '@microsoft/bf-dialog/lib/library/schemaMerger';
 
 const API_ROOT = '/api';
 
+
+const normalizeFeed = (feed) => {
+  if (feed.objects) {
+    // this is an NPM feed
+    return feed.objects.map((i) => {
+      return {
+      name: i.package.name,
+      version: i.package.version,
+      authors: i.package?.author?.name,
+      keywords: i.package.keywords,
+      repository: i.package?.links.repository,
+      description: i.package.description,
+      language: 'js',
+      source: 'npm',
+    }});
+
+  } else if (feed.data) {
+    // this is a nuget feed
+    return feed.data.map((i) => { return {
+      name: i.id,
+      version: i.version,
+      authors: i.authors[0],
+      keywords: i.tags,
+      repository: i.projectUrl,
+      description: i.description,
+      language: 'c#',
+      source: 'nuget',
+    }});
```

Comment:
Nit: this is formatted strangely. Can you apply prettier to this (an the ui files) please?

Reply:
I applied the same eslintrc rules from the main composer project. good idea.